### DISCUSSION TITLE
Regular meeting for managing pull requests

### DISCUSSION THREAD

**Discussion by oyvindseland** [NorESMhub/CAM#64] [Team Posts]
Hi 
There is a suggestion and I think we also discussed it at some point to have biweekly meetings to decide on pull requests and other discussions on CAM6-Nor github tecnincal issues. Since the idea is to have Oslo-Bergen telecons every other week I suggest that we use the alternative weeks for github and version discussion. The meetings should be shorter though. I suggest we start next Wednesday to agree on the format and on the status on model documentation on github

**Comment by oyvindseland** [comment]
Michael reminded me that there is a meeting next week. I suggest that a meeting in the CAM core group is then scheduled the week after, i.e 10th June, 13-14

**Comment by tto061** [comment]
I agree

Thomas

**Comment by DirkOlivie** [comment]
Ok for me.

Dirk

**Comment by MichaelSchulzMETNO** [comment]
fine with me!

**Comment by oyvindseland** [comment]
Reorganized through CAM-Core

Closing this issue
